### ISSUE TITLE
Info Request on how to create an SPK file from this repo

### ISSUE THREAD

**Issue by WizBangCrash** [itn3rd77/openhab-synology-spk#3]
Hi,

I came across your project this morning and having looked at the source I would rather use this package for my system than the other openHAB sync repos in GitHub.
So now I need to work out how to build the SPK and wondered if you could give me a pointer or two.

I am assuming I need to follow the instructions on how to build a package form here: https://developer.synology.com/developer-guide/create_package/index.html

Are their any pointers/tips I need to be aware of to set up a build environment on my DS716+II running DSM 6.1?

**Comment by itn3rd77** [comment]
Hi,

I've setup a Docker container (Debian) for the build process (aka Synology Toolkit). Be aware that the Docker container has to run with high privileges! Then follow the instruction to _"Install Toolkit"_ and _"Prepare Build Environment"_. Check out the branch you want to build the package to the _"source"_ folder. 

To build the package run the following command:
```
./pkgscripts/PkgCreate.py -x0 -c openHAB-2.2 -S
```

Where _openHAB-2.2_ refers to the folder inside the source directory where you checked out he sources from openhab-synology-spk.

If you don't mind building the packages from source you can download the latest packages from the link I have added to the project site. 

Have fun!

Regards
Ingo

**Comment by WizBangCrash** [comment]
Thanks Ingo.
For now, I will download and use the package from your link as I haven't started using Docker yet and I don't want to get too distracted away form setting up my openHAB server :-)

FYI I tried adding package source using the http://spk.i-matrixx.de/ URL, but Dam says it's an "invalid location", so I downloaded the package and it installed with no issues.

One request I would like to make is to have the "openHAB/src/userdata/logs" folder linked on the openHAB share as it will make viewing the logs a lot easy from vscode if I can see them in the openHAB share.

Thank you.
